express-get-json/index.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+
+const app = express();
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.listen(3000);
+
+app.get('/api/grades', (req, res) => {
+  const resArr = [];
+  for (const prop in grades) {
+    const resJSON = grades[prop];
+    resArr.push(resJSON);
+  }
+  res.json(resArr);
+});
```

Comment:
Since servers can take a long time to start up, it's generally a good idea to start listening _after_ all the routes have been loaded, and print out a message when the server starts listening so you know when it's ready to take requests.